code/datums/martial/holy_crunch.dm

```diff
@@ -0,0 +1,40 @@
+/datum/martial_art/holy_crunch
+	name = "Holy Crunch"
+	deflection_chance = 5
+	no_guns = TRUE
+	allow_temp_override = FALSE
+
+/datum/martial_art/chaplain_snap/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("power word: grail", "power word: justice", "power word: purge", "power word: judicator", "power word: banish")
+	D.visible_message("<span class='danger'>[A] uses [atk_verb] on [D]!</span>", \
+					  "<span class='userdanger'>[A] uses [atk_verb] on you!</span>")
+	D.apply_damage(10, BRUTE)
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 25, 1, -1)
+	if(prob(D.getBruteLoss()) && !D.lying)
```

Comment:
first attack has a 10% chance to knock down, with each attack increasing it by 10%

Reply:
that's a good comment to add

Reply:
uh huh!